prometheus_client/core.py

```diff
@@ -183,6 +183,24 @@ def collect():
 
 @_MetricWrapper
 class Counter(object):
+    '''
```

Comment:
Multi-line docstrings should have a single line at the very start, so move the line below up

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings